docs/docs/events/builtin/extra/chmod_common.md

```diff
@@ -0,0 +1,34 @@
+# chmod_common
```

Comment:
I don't think the name chmod_common is so clear to the user.
A better name for this event may be `file_chmod`